lib/knife-zero/core/bootstrap_context.rb

```diff
@@ -1,80 +1,80 @@
 require 'chef/knife/core/bootstrap_context'
-require "knife-zero/helper"
+require 'knife-zero/helper'
 
 class Chef
   class Knife
     module Core
       class BootstrapContext
-         class_eval do
-           alias :orig_validation_key validation_key
-           def validation_key
-             if @chef_config[:knife_zero]
-               OpenSSL::PKey::RSA.new(2048).to_s
-             else
-               orig_validation_key
-             end
-           end
+        class_eval do # rubocop:disable Metrics/BlockLength
+          alias_method :orig_validation_key, :validation_key
+          def validation_key
+            if @chef_config[:knife_zero]
+              OpenSSL::PKey::RSA.new(2048).to_s
+            else
+              orig_validation_key
+            end
+          end
 
-           alias :orig_config_content config_content
-           def config_content
-             client_rb = orig_config_content
-             white_lists = []
-             %w{ automatic_attribute_whitelist default_attribute_whitelist normal_attribute_whitelist override_attribute_whitelist }.each do |white_list|
-               if Chef::Config[:knife][white_list.to_sym] && Chef::Config[:knife][white_list.to_sym].is_a?(Array)
-                 white_lists.push([
-                   white_list,
-                   Chef::Config[:knife][white_list.to_sym].to_s
-                 ].join(" "))
-               end
-             end
-             client_rb << white_lists.join("\n")
+          alias_method :orig_config_content, :config_content
+          def config_content # rubocop:disable Metrics/AbcSize
+            client_rb = orig_config_content
+            white_lists = []
+            %w{automatic_attribute_whitelist default_attribute_whitelist normal_attribute_whitelist override_attribute_whitelist}.each do |white_list|
+              next unless Chef::Config[:knife][white_list.to_sym] && Chef::Config[:knife][white_list.to_sym].is_a?(Array)
```

Comment:
__[RuboCop]__ Style/SafeNavigation: Use safe navigation (`&.`) instead of checking if an object exists before calling the method. ([link](https://sider.review/gh/repos/23179315/pulls/125#issue-4173575))

<sub>posted by Sider</sub>